inbox/events/google.py

```diff
@@ -131,8 +132,9 @@ def _get_raw_events(
         url = "https://www.googleapis.com/calendar/v3/calendars/{}/events".format(
             urllib.parse.quote(calendar_uid)
         )
+        params = {"eventTypes": "default"}
         try:
-            return self._get_resource_list(url, updatedMin=sync_from_time_str)
+            return self._get_resource_list(url, updatedMin=sync_from_time_str, **params)
```

Comment:
Why the use of kwargs? Is it because the linter would complain about passing `eventTypes="default"` explicitly to the function call?

Reply:
That was indeed unnecessary, I changed it.